app/services/analytics_events.rb

```diff
@@ -1374,7 +1374,6 @@ def idv_doc_auth_submitted_pii_validation(
   # @param analytics_id [String] "Doc Auth" for remote unsupervised, "In Person Proofing" for IPP
   # @param errors [Hash] Details about vendor-specific errors encountered during the stages of the identity resolution process
   # @param flow_path [String] "hybrid" for hybrid handoff, "standard" otherwise
-  # @param irs_reproofing [Boolean] Whether the user is being forced to reproof to access the IRS
```

Comment:
This was from back in the Team Agnes Attempts API days. Most of that code has already been ripped out; this one must have escaped notice.